fleet/launching-containers-fleet.md

```diff
@@ -295,7 +295,7 @@ MachineMetadata=region=east
 
 ## Running fleet via a Container Linux Config
 
-On Container Linux, only fleet 0.11.x is available under /usr/bin. That one might be obsolete for users who want to try out more recent versions. In that case, users should define their own custom Container Linux Config to be able to run any version of fleet as they want. For example:
+A fleet binary is no longer installed since Container Linux 1675.0.0. To continue running fleet, users should define their own container-based service and PolicyKit rule. For example:
```

Comment:
Might as well call it 1675.0.1, since 0.0 was never released.

This section should really be moved toward the top of the file, since we're currently telling people how to use a tool, then mentioning afterward how to install it.